File: KULIAH/LogCat/app/src/main/java/id/ac/unsyiah/android/logcat/MainActivity.java
package id.ac.unsyiah.android.logcat;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String tes = "Log Cat";

        int nilai = 10;
        String nilaiString = Integer.toString(nilai);

        Log.d("MainActivity","onCreate");
    }
}

File: Kotlin/SafeRoute/app/src/main/java/com/bnawan/saferoute/MainActivity.kt
package com.bnawan.saferoute

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import androidx.navigation.NavController
import androidx.navigation.findNavController
import androidx.navigation.ui.NavigationUI
import com.bnawan.saferoute.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding:ActivityMainBinding
    private lateinit var navController: NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_main)

        navController = this.findNavController(R.id.navigation_host)
        NavigationUI.setupWithNavController(binding.bottomNavigation, navController)
    }
}